platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/turtle/TurtleNodeConfiguration.java

```diff
@@ -57,26 +63,6 @@ Configuration createConfiguration() {
                 .getOrCreateConfig();
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    @NonNull
-    public TurtleNodeConfiguration set(@NonNull final String key, final boolean value) {
-        overriddenProperties.put(key, Boolean.toString(value));
-        return this;
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    @NonNull
-    public TurtleNodeConfiguration set(@NonNull final String key, @NonNull final String value) {
-        overriddenProperties.put(key, value);
-        return this;
-    }
-
```

Comment:
I do not recall why it was done like this. Probably simply copied from the PoC. I think we can move the content of `createBasicConfigBuilder()` into `createConfiguration()`.

As we do not store the version in the configuration anymore, I think the `SemanticVersionConverter` can be removed.